docker/federation/docker-compose.yml

```diff
@@ -29,7 +29,7 @@ services:
       - ./volumes/pictrs_alpha:/mnt
 
   lemmy-alpha-ui:
-    image: dessalines/lemmy-ui:v0.0.14
+    image: dessalines/lemmy-ui:dev
```

Comment:
This should be the normal release (I think I left this in).

Reply:
Ah yeah, I see that now. It'll replace this the next prod deploy. This was necessary to test this PR tho, because the v0.0.14 didn't work with it.